addons/web/static/src/views/fields/properties/properties_field.scss

```diff
@@ -1,5 +1,8 @@
 .o_field_properties {
     width: 100%;
+    // be sure that the property field can take zero height
+    // in case no property exists
+    margin-bottom: 0 !important;
```

Comment:
I don't think, because properties field and not added inside a `<group/>` but it simulate `<group/>` (if you see what I mean)

![screenshot_spect](https://user-images.githubusercontent.com/54316226/233277800-e8187936-dc37-42b7-906c-e8ce2413c2f8.png)